KoreBuild-dotnet/build/_dotnet-test.shade

```diff
@@ -18,7 +18,16 @@ default configuration = 'Debug'
 
 @{
     var projectFolder = Path.GetDirectoryName(projectFile);
```

Comment:
I believe dotnet test flows any unrecognized arguments to the testrunner. e.g. `--configuration Debug` is handled by `dotnet-test.exe` but `-parallel none` is passed to `dotnet-test-xunit.exe`. Order matters though: cref https://github.com/dotnet/cli/issues/1456